week_4/quiz-trivia-app/src/app/screens/finish-page/statistics/statistics.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import { QuizService } from '../../../data/quiz.service';
+import { Quiz } from '../../../data/quiz.model';
+import { StorageService } from '../../../storage.service';
+
+@Component({
+  selector: 'app-statistics',
+  standalone: true,
+  imports: [],
+  templateUrl: './statistics.component.html',
+  styleUrl: './statistics.component.css',
+})
+export class StatisticsComponent implements OnInit {
```

Comment:
this component sounds more like resultComponent rather than statistics.

Reply:
renamed